send/generic_sender.py

```diff
@@ -107,6 +109,7 @@ def prepare_ssh_transport_command():
 		if destination_type == send_lib.DESTINATION_TYPE_EMAIL or destination_type == send_lib.DESTINATION_TYPE_SERVICE_SPECIFIC:
 			print("Destination type " + destination_type + " is not supported yet.", file=sys.stderr)
 			exit(1)
+	auth_token = sys.argv[4]
```

Comment:
This doesn't work since we check number of input params in `send_lib.check_input_fields(sys.argv)` a few lines above and its implementation needs to be updated for this new optional param.